MCP_Server/tools/devices.py

```
@@ -0,0 +1,195 @@
+                "parameter_index": parameter_index,
+                "value": value,
+                "track_type": track_type,
+            })
+            return f"Set device {device_index} parameter {parameter_index} to {value}"
+        except Exception as e:
+            logger.error(f"Error setting device parameter: {str(e)}")
+            return f"Error setting device parameter: {str(e)}"
+
+    @mcp.tool()
+    def get_rack_device_info(
+        ctx: Context,
```

Ruff (0.15.1)

[warning] Unused function argument: `ctx`

(ARG001)